fjord/feedback/tests/test_views.py

```diff
@@ -102,6 +102,37 @@ def test_valid_sad(self):
         eq_(u'Firefox', feedback.product)
         eq_(u'14.0.1', feedback.version)
 
+    @with_waffle('thankyou', True)
+    def test_use_thank_you_sad(self):
+        """Thanks_sad template is used when appropriate"""
+        url = reverse('feedback', args=(u'firefox',))
```

Comment:
I just updated the reverse function:

https://github.com/mozilla/fjord/blob/d3fe0fcb975f7a81209b5e7d05d7f87e743f1a83/fjord/base/urlresolvers.py#L25

That change allows you to provide a `locale` argument to explicitly set the locale to use in the url. This line should explicitly set the locale to `en-US`.